packages/iracing-plugin-mirabox/rollup.config.mjs

```
@@ -15,6 +16,72 @@ const actionsPackagePath = path.resolve(__dirname, "../iracing-actions/src");
+ * The merged object feeds `@rollup/plugin-replace` (compile-time constants),
+ * `piTemplatePlugin` (EJS `platform` variable), and the emitted `config.json`.
+ */
+const platformFeaturesPath = path.resolve(__dirname, "platform-features.json");
+const localFeaturesPath = path.resolve(__dirname, "../../feature-flags.local.json");
+const committedFeatures = JSON.parse(readFileSync(platformFeaturesPath, "utf-8"));
+let platformFeatures = committedFeatures;
+if (existsSync(localFeaturesPath)) {
+	const localFeatures = JSON.parse(readFileSync(localFeaturesPath, "utf-8"));
+	const { known, unknown } = partitionOverride(committedFeatures, localFeatures);
+	if (unknown.length > 0) {
+		console.warn(
```

ESLint

[error] 'console' is not defined.

(no-undef)